src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -165,222 +150,21 @@ export const DatasetSearchTable = (props) => {
     };
     search();
   };
-
-
-
-  const selectHandler = async (event, data, selector) => {
-    let idsToModify = [];
-    if (selector === 'all') {
-      data.rows.forEach((row) => {
-        row.subtable.rows.forEach((subRow) => {
-          idsToModify.push(subRow.id);
-        });
-      });
-    } else if (selector === 'row') {
-      const rowIds = data.subtable.rows.map(row => row.id);
-      const isRowSelected = rowIds.every(id => selected.includes(id));
-      isRowSelected ?
-        await Metrics.captureEvent(eventList.dataLibrary, {'action': 'study-unselected'}) :
-        await Metrics.captureEvent(eventList.dataLibrary, {'action': 'study-selected'});
-      data.subtable.rows.forEach((row) => {
-        idsToModify.push(row.id);
-      });
-    } else if (selector === 'subrow') {
-      selected.includes(data.id) ?
-        await Metrics.captureEvent(eventList.dataLibrary, {'action': 'dataset-unselected'}) :
-        await Metrics.captureEvent(eventList.dataLibrary,{'action': 'dataset-selected'});
-      idsToModify.push(data.id);
-    }
-
-    let newSelected = [];
-    const allSelected = idsToModify.every((id) => selected.includes(id));
-    if (allSelected) {
-      newSelected = selected.filter((id) => !idsToModify.includes(id));
-    } else {
-      newSelected = selected.concat(idsToModify);
-    }
-
-    setSelected(newSelected);
-  };
-
-  const getExportableDatasets = async (event, data) => {
-    setTdrApiUrl(await Config.getTdrApiUrl());
-    // Note the dataset identifier is in each sub-table row.
-    const datasetIdentifiers = data.subtable.rows.map((row) => row.datasetIdentifier);
-    const snapshots = await TerraDataRepo.listSnapshotsByDatasetIds(datasetIdentifiers);
-    if (snapshots.filteredTotal > 0) {
-      const datasetIdToSnapshot = _.chain(snapshots.items)
-      // Ignore any snapshots that a user does not have export (steward or reader) to
-        .filter((snapshot) => _.intersection(snapshots.roleMap[snapshot.id], ['steward', 'reader']).length > 0)
-        .groupBy('duosId')
-        .value();
-      setExportableDatasets(datasetIdToSnapshot);
-    }
-  };
-
-  const expandHandler = async (event, data) => {
-    try {
-      getExportableDatasets(event, data);
-    } catch {
-      Notifications.showError({ text: 'Unable to retrieve exportable datasets from Terra' });
-    }
-  };
-
-  const collapseHandler = () => {
-    setExportableDatasets({});
-  };
-
   const applyForAccess = async () => {
-    const draftDatasets = selected.map((id) => parseInt(id.replace('dataset-', '')));
-    const darDraft = await DAR.postDarDraft({ datasetId: draftDatasets });
+    const darDraft = await DAR.postDarDraft({ datasetId: selected });
     history.push(`/dar_application/${darDraft.referenceId}`);
   };
 
   const clearSearchRef = () => {
     searchRef.current.value = '';
     filterHandler(null, datasets, '', '');
   };
-
-  const openTranslatedDUL = (dataUse) => {
```

Comment:
This was unused in the new version - I figure we can add it back if need be. Not sure because I didn't see dataUse in the mocks: https://www.figma.com/design/MkffpVbUdP4lWvkN9OWiro/Data-Library?node-id=23-43&node-type=frame

Reply:
Agreed - I'm usually in favor of aggressive pruning, it's easy enough to re-add. Especially for anything related to data use translations, we have some very inefficient code related to that functionality. If we do need it, we need to re-evaluate how we do it.